projects/plugins/jetpack/_inc/client/components/navigation/index.jsx

```diff
@@ -104,7 +106,28 @@ export class Navigation extends React.Component {
 							onClick={ this.trackRecommendationsClick }
 							selected={ this.props.location.pathname.startsWith( '/recommendations' ) }
 						>
-							{ _x( 'Recommendations', 'Navigation item.', 'jetpack' ) }
+							{ createInterpolateElement(
+								sprintf(
+									/* translators: %s is a count of how many new (unread) recommendations are available. */
+									_x(
+										'Recommendations <CountWrap><span>%d</span></CountWrap>',
+										'Navigation item.',
+										'jetpack'
+									),
+									this.props.newRecommendationsCount
+								),
+								{
+									CountWrap: (
+										<span
+											className={
+												'dops-section-nav-tab__update-badge count-' +
+												this.props.newRecommendationsCount
+											}
+										></span>
+									),
+									span: <span className="update-count"></span>,
```

Comment:
That's a lot of span. :) I assume it wasn't possible to do without the extra span?